src/BOCS/compute_pressure_bocs.cpp

```diff
@@ -139,7 +139,8 @@ void ComputePressureBocs::init()
   if (keflag) {
     temperature = modify->get_compute_by_id(id_temp);
     if (!temperature)
-      error->all(FLERR,"Could not find compute pressure/bocs temperature compute {}", id_temp);
+      error->all(FLERR, Error::NOLASTLINE,
```

Comment:
I hadn't noticed the addition of `Error::NOLASTLINE` before, does its inclusion do anything different than the regular `error->all` calls w/o the argument?

Reply:
The purpose is to suppress a "Last command: run ###" message that is appended by default (for backward compatibility and corresponds to `Error::NOPOINTER`, i.e print  the last command, but do not point to any keyword in it) and can be confusing for all errors happening in the "middle" or a run, e.g. during setup() or init().
I've tried to explain the current status here:
https://docs.lammps.org/Developer_notes.html#errors-warnings-and-informational-messages

Reply:
Ah, that makes sense. I had picked up on the new use of arg indices before but not that.